scst-store/cluser-specific-scanner-configurations.hbs.md

```diff
@@ -0,0 +1,42 @@
+# Connecting Vulnerability scanning to SCST - Store 
```

Comment:
Change title:  Connecting vulnerability scanning to Supply Chain Security Tools - Store